### DISCUSSION TITLE
Improvement: Implement `STAT_TO_ALLOCATE = "auto"`

### DISCUSSION THREAD

**Discussion by ramotar** [bumbleshoot/automate-habitica#23] [Suggestions]
**What is the suggestion?**
Provide an automatic selection mode for `STAT_TO_ALLOCATE`, that allocates stat points depending on the current class of the user.

**Additional context**
Every class has a different main stat, that its skills are based on. When changing classes, one must remember to change the configuration of Automate Habitica. It would be quite easy to implement an automatic mode, that simply allocates the main stat for the current class of the user.

**Script settings**
The settings could then look like this:
```javascript
const STAT_TO_ALLOCATE = "auto"; // str, int, con, per, auto
```

**Comment by ramotar** [comment]
@bumbleshoot Sorry for making this a bug report on the first try!

**Comment by bumbleshoot** [comment]
Hi @ramotar, thanks for the suggestion! Copying & pasting my reply to your pull request here:

There are lots of ways users might choose to allocate their stats - allocating all stat points to the primary stat is not always necessarily the best strategy. It depends on the user's play style. Someday, I would like to give users more control over stat allocation (ie. rather than all stat points going to one stat, the user can distribute them automatically to all four stats however they want). This is on my list of "nice to have" features that I will build someday if I have time. :)
